prog/storage/setup_spdk.rb

```diff
@@ -62,6 +64,10 @@ def self.assemble(vm_host_id, version, start_service: false, allocation_weight:
       vm_host_id: vm_host.id
     ).update(allocation_weight: frame["allocation_weight"])
 
+    if frame["start_service"]
+      VmHost.where(id: vm_host.id).update(used_hugepages_1g: Sequel[:used_hugepages_1g] + 1)
```

Comment:
This looks pretty fragile & expedient (which I can appreciate), but, we should probably re-run "learn" code to square up clover's understanding of a computer in such a situation. But currently, our programming model for that is not so terse.

Reply:
Re-running learn while VMs can be provisioned can have concurrency issues, unless we do some locking.

E.g.
1. Get hugepages info
2. A VM is provisioned
3. Update hugepages info => Ignores the hugepages used by the new VM.

Reply:
Since relearning without running into concurrency issues is not straightforward, I'm keeping it for a separate PR.

Reply:
Yes, that's a good point. I toyed with the ugly idea of teaching Learn code how to do updates on a target record (even postgres can benefit from hugepage awareness), but the concurrency issue does make that not as appealing, since we'd probably want to synchronize it through the HostNexus in this case.